### PR TITLE
Responsive-TableV2

### PR DESCRIPTION
**Pull request by kittyevi**
PR: #2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+<html lang="de">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="style.css" rel="stylesheet">
+    <script src="script.js"></script>
+    <title>responsive-table</title>
+  </head>
+  <body>
+    <div class="body__table-container">
```
Comment (DevBastiS) [inline index.html]:
```suggestion
    <div class="table-container">
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      <table id="body__table-id" class="table-container__table">
```
Comment (DevBastiS) [inline index.html]:
```suggestion
      <table id="table-id" class="table-container__table">
```

File: script.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+function hideTableContents(className) {
```
Comment (DevBastiS) [inline script.js]:
```suggestion
function hideColumnContents(className) {
```

File: script.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  const allCellsInClass = document.getElementsByClassName(className);
+
+  for (let cellIterator = 0; cellIterator < allCellsInClass.length; cellIterator++) {
```
Comment (DevBastiS) [inline script.js]: Do this with a for of loop.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of?retiredLocale=de

**Review by DevBastiS — CHANGES_REQUESTED** [review]

File: style.css [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.table-container__table th:hover {
+  cursor: pointer;
+}
+
+.table-container__table td {
+  border-right: 1px solid black;
+}
```
Comment (MKrips) [inline style.css]: you can you use classes you gave your elements for this or create a new class (one for header and one for cells) and style them

**Review by MKrips — APPROVED** [review]

**Review by DevBastiS — APPROVED** [review]

**Pull request merged**